scripts/editor/icons/story.js

```diff
@@ -24,50 +25,66 @@ const IconsOutput = (props) => {
 				css={{
 					fontSize: '30px',
```

Comment:
This is in `px` intentionally?

Reply:
Whoops forgot to change it, that's how it was originally. Thanks!